cherab/openadas/openadas.py

```diff
@@ -134,20 +134,20 @@ def beam_cx_pec(self, donor_ion, receiver_ion, receiver_charge, transition):
         if isinstance(donor_ion, Isotope):
             donor_ion = donor_ion.element
 
-        if isinstance(receiver_ion, Isotope):
-            receiver_ion = receiver_ion.element
+        receiver_ion_element = receiver_ion.element if isinstance(receiver_ion, Isotope) else receiver_ion
 
         try:
             # read data
-            wavelength = repository.get_wavelength(receiver_ion, receiver_charge - 1, transition, repository_path=self._data_path)
-            data = repository.get_beam_cx_rates(donor_ion, receiver_ion, receiver_charge, transition,
+            data = repository.get_beam_cx_rates(donor_ion, receiver_ion_element, receiver_charge, transition,
```

Comment:
Also here, could you please add a line comment saying the code is getting the element's data, because adas doesn't have isotope data?